docker-compose.yml

```diff
@@ -129,5 +129,30 @@ services:
                 aliases:
                     - mongo-useradm
 
+    mender-conductor:
```

Comment:
what do you think about having conductor in separate compose file similarly to minio+nginx ?